### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SaintPeter**
I've done a first pass on cleaning up this doc.  It probably needs further work, but this is a good start.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
-7.  Once your code is ready, submit a [pull request](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Pull-Request-Contribute) from your branch to Free Code Camp's `staging` branch. We'll do a quick code review and give you feedback, then iterate from there. It may also be helpful to read about git [rebasing](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase).
+2.  Pick an issue that nobody has claimed and start working on it. If your issue isn't on the board, open an issue. If you think you can fix it yourself, start working on it. Feel free to ask for help in our [General Gitter Chat Room](https://gitter.im/FreeCodeCamp/FreeCodeCamp), or in our [Help Contributors Chat Room](https://gitter.im/FreeCodeCamp/HelpContributors?utm_source=share-link&utm_medium=link&utm_campaign=share-link)
+3. You can find issues we are seeking assistance on by searching for the [Help Wanted](https://github.com/FreeCodeCamp/FreeCodeCamp/labels/help%20wanted) tag.
```
Comment (bugron) [inline CONTRIBUTING.md]: I think we need line breaks here:

**Comment by SaintPeter** [comment]
Fixed Markdown.

**Comment by bugron** [comment]
Looks good, waiting for @ltegman to QA.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
-7.  Once your code is ready, submit a [pull request](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Pull-Request-Contribute) from your branch to Free Code Camp's `staging` branch. We'll do a quick code review and give you feedback, then iterate from there. It may also be helpful to read about git [rebasing](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase).
+2.  Pick an issue that nobody has claimed and start working on it. If your issue isn't on the board, open an issue. If you think you can fix it yourself, start working on it. Feel free to ask for help in our [General Gitter Chat Room](https://gitter.im/FreeCodeCamp/FreeCodeCamp), or in our [Help Contributors Chat Room](https://gitter.im/FreeCodeCamp/HelpContributors?utm_source=share-link&utm_medium=link&utm_campaign=share-link)
```
Comment (ltegman) [inline CONTRIBUTING.md]: Remove query string with `utm` parameters from URL.

**Comment by SaintPeter** [comment]
Fixed URL, should be good to go.

**Comment by ltegman** [comment]
Awesome! :+1:

**Pull request merged**